src/components/v2/V2Project/V2FundingCycleSection/PayoutSplitsCard.tsx

```diff
@@ -64,6 +64,32 @@ export default function PayoutSplitsCard({
   const hasDuration = fundingCycleDuration?.gt(0)
   const canEditPayouts = useHasPermission(V2OperatorPermission.SET_SPLITS)
 
+  const effectiveDistributionLimit = distributionLimit ?? BigNumber.from(0)
+  const distributedAmount = usedDistributionLimit ?? BigNumber.from(0)
+
+  const distributable = effectiveDistributionLimit.sub(distributedAmount)
+
+  const distributableAmount = balanceInDistributionLimitCurrency?.gt(
+    distributable,
+  )
+    ? distributable
+    : balanceInDistributionLimitCurrency
+
+  const distributeButtonDisabled = isPreviewMode || distributableAmount?.eq(0)
+
+  function DistributeButton(): JSX.Element {
+    return (
+      <Button
+        type="ghost"
+        size="small"
+        onClick={() => setDistributePayoutsModalVisible(true)}
+        disabled={distributeButtonDisabled}
+      >
+        <Trans>Distribute funds</Trans>
+      </Button>
+    )
+  }
+
```

Comment:
This might fix the div situation.

This way we accept all "standard" props that HTMLButtonElement accepts, which includes the ones Tooltips need.

```suggestion
  function DistributeButton(props: React.HTMLProps<HTMLButtonElement>): JSX.Element {
    return (
      <Button
        type="ghost"
        size="small"
        onClick={() => setDistributePayoutsModalVisible(true)}
        disabled={distributeButtonDisabled}
        {...props}
      >
        <Trans>Distribute funds</Trans>
      </Button>
    )
  }

```

Reply:
Antd `<Button/>` can't take `React.HTMLProps<HTMLButtonElement>` as props. I typed the `props` to antd's `ButtonProps` but still getting the same problem without the `<div/>`s

Reply:
Eh, couldnt get it to work either.

No big.